docs/tutorials/simple_trading.py.md

```diff
@@ -84,11 +84,11 @@ class SimpleMarketListener(MarketStreamListener):
     futures = [instrument for instrument in instrument_data['data'].values() if instrument['type'] == 'futures'][0]
     selected_futures_id = futures['instrument_id']
 
-  def on_order_book(self, order_book):
+  def on_order_book(self, order_book):  
     if order_book['instrument_id'] != selected_futures_id:
       return 
     bids = order_book['bids']
-    if bids and (not bids[0] or bids[0][0] > sell_threshold):
+    if bids and (not bids[0] or float(bids[0][0]) > sell_threshold):
```

Comment:
Ah OK. I get your intention now. And while I entirely agree with you on the principles, if this tutorial aims to teach about defensive programming, it need to be more explicit (e.g. logging that we received garbage from the server).